src/cardSystem/Card.py

```diff
@@ -35,5 +35,9 @@ def render(self, screen, order):
         # Draw the border around the card
         pygame.draw.rect(screen, (0, 0, 0), (start_x, start_y, CARD_WIDTH, CARD_HEIGHT), 1)
 
+        
+    def renderPosition(self, screen, position, scale):
```

Comment:
@rymTooo So, order(position) of the cards in hand will use this function instead?

Reply:
Could be. The old method should be able to use the new one.